lux/_config/config.py

```diff
@@ -83,6 +85,23 @@ def sort(self, flag: Union[str]):
                 stacklevel=2,
             )
 
+    @property
```

Comment:
Not sure if sort makes sense as a global option, since it is more common to do one type of sorting for an action, but a different sort mechanism for another action.

Reply:
I added two implementations -- one global ordering (this could be useful for sorting by attribute name or something really general that you want to standardize across all actions) and an action-wise ordering dictionary. If no ordering is defined for the specific action, then we fallback to the global ordering.

Reply:
This sounds great, let's add some docs that explain the two different types of ways to specify sorting!